io/io/src/TDirectoryFile.cxx

```diff
@@ -874,17 +874,16 @@ TObject *TDirectoryFile::FindObjectAny(const char *aname) const
 /// |  foo;1  | get cycle 1 of foo on file |
 ///
 /// The retrieved object should in principle derive from TObject.
-/// If not, the function TDirectoryFile::GetObject should be called.
+/// If not, the function TDirectoryFile::Get<T> should be called.
 /// However, this function will still work for a non-TObject, provided that
 /// the calling application cast the return type to the correct type (which
 /// is the actual type of the object).
 ///
-/// ### The GetObjectMethod
-/// The method GetObject offers better protection and avoids the need
-/// for any cast:
+/// ### The Get<T> Method
+/// The method Get<T> offers better protection and avoids the need for any
+/// cast:
 /// ~~~{.cpp}
-/// MyClass *obj;
-/// directory->GetObject("some object",obj);
+/// auto obj = directory->Get<MyClass>("some object");
```

Comment:
Simply using `auto ptr` will probably be clear enough that this returns a pointer.

Reply:
I like it! I'm using `auto objPtr` instead though since that notation was already established in the rest of that documentation section.